### DISCUSSION TITLE
Can Tauri be Used for Building a Grammarly-Like Windows Application?

### DISCUSSION THREAD

**Discussion by MahdiTa97** [tauri-apps/tauri#8617] [General]
Hi Tauri community!

I'm exploring the possibility of using Tauri to build a Windows application with functionalities similar to **Grammarly**. However, I'm primarily focused on the client-side aspects and not the complex NLP and language processing.

My key requirements include sending user input to a server and receiving a JSON response with various data. Additionally, I'm interested in challenges related to implementing a gadget icon and reading user input seamlessly.

Has anyone attempted something similar with Tauri? What specific challenges should I be aware of, especially regarding UI integration and capturing user input?

I'd appreciate any insights, suggestions, or experiences you might have on this topic. Thank you!

**Comment by ilyaLibin** [comment]
+1 for the question. How do they add it to every textarea? and not only in browser? Also in Slack and other native apps.
